src/test/java/org/kcctl/command/GetPluginsCommandTest.java

```diff
@@ -37,33 +40,61 @@
+                "source   io.debezium.connector.db2.Db2Connector                      " + debeziumVersion,
+                "source   io.debezium.connector.mongodb.MongoDbConnector              " + debeziumVersion,
+                "source   io.debezium.connector.mysql.MySqlConnector                  " + debeziumVersion,
+                "source   io.debezium.connector.oracle.OracleConnector                " + debeziumVersion,
+                "source   io.debezium.connector.postgresql.PostgresConnector          " + debeziumVersion,
+                spannerConnectorDescription,
+                "source   io.debezium.connector.sqlserver.SqlServerConnector          " + debeziumVersion,
+                "source   io.debezium.connector.vitess.VitessConnector                " + debeziumVersion,
+                fileConnectorDescription,
+                "source   org.apache.kafka.connect.mirror.MirrorCheckpointConnector   " + mirrorMakerVersion,
+                "source   org.apache.kafka.connect.mirror.MirrorHeartbeatConnector    " + mirrorMakerVersion,
+                "source   org.apache.kafka.connect.mirror.MirrorSourceConnector       " + mirrorMakerVersion)
+                .filter(Objects::nonNull)
+                .toArray(String[]::new);
+
         context.runAndEnsureExitCodeOk("--types=source");
         assertThat(context.output().toString().trim().lines())
                 .map(String::trim)
-                .containsExactly(
-                        "TYPE     CLASS                                                       VERSION",
-                        "source   io.debezium.connector.db2.Db2Connector                      " + debeziumVersion,
-                        "source   io.debezium.connector.mongodb.MongoDbConnector              " + debeziumVersion,
-                        "source   io.debezium.connector.mysql.MySqlConnector                  " + debeziumVersion,
-                        "source   io.debezium.connector.oracle.OracleConnector                " + debeziumVersion,
-                        "source   io.debezium.connector.postgresql.PostgresConnector          " + debeziumVersion,
-                        "source   io.debezium.connector.spanner.SpannerConnector              " + debeziumVersion,
-                        "source   io.debezium.connector.sqlserver.SqlServerConnector          " + debeziumVersion,
-                        "source   io.debezium.connector.vitess.VitessConnector                " + debeziumVersion,
-                        "source   org.apache.kafka.connect.mirror.MirrorCheckpointConnector   " + kafkaVersion,
-                        "source   org.apache.kafka.connect.mirror.MirrorHeartbeatConnector    " + kafkaVersion,
-                        "source   org.apache.kafka.connect.mirror.MirrorSourceConnector       " + kafkaVersion);
+                .containsExactly(expectedOutput);
     }
 
     @Test
+    @SkipIfConnectVersionIsOlderThan("3.2")
```

Comment:
This syntax is so smooth and unobtrusive. Great idea, Gunnar!

Reply:
Ha yeah, thanks a lot for setting this up! We have something similar in Debezium, worked great there as well.